src/internal/internalAPIs.ts

```diff
@@ -235,7 +235,7 @@ export function handleParentMessage(evt: DOMMessageEvent): void {
     const message = evt.data as MessageResponse;
     const callback = GlobalVars.callbacks[message.id];
     if (callback) {
-      callback.apply(null, message.args);
+      callback.apply(null, [...message.args, message.isPartialResponse]);
```

Comment:
Why is this change necessary? Seems weird that we would be passing this property to the callback? I don't think this is something a callback should ever have to handle itself.

Reply:
Its used specifically for this part here where we proxy the response from Teams to the current frame to the child frame that called the API https://github.com/OfficeDev/microsoft-teams-library-js/pull/483/files#diff-fb785af0e32520a03c689ec3c20c481d2c9b5d362ef181402123f2b50be7a418R277

I don't really see any other better way to pass `isPartialResponse` to reconstructing the proxy partial response to the child frame so that's why this change is necessary.

Reply:
One thing we can do to be safer is to only pass `isPartialResponse` to the callback args if the callback is specificially the proxy callback defined here https://github.com/OfficeDev/microsoft-teams-library-js/pull/483/files#diff-fb785af0e32520a03c689ec3c20c481d2c9b5d362ef181402123f2b50be7a418L274 
(i.e we can change it from an anonymous function to a named one, and check the callbacks name).
But either way, given how the current code works, passing this property as an arg to the callback seems to be the only way to solve this issue.

Reply:
I'm fine with either option; I would say do what you think is best. FYI @sachingupta